### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Nathan Wulliamoz & Benjamin Kocher

### DISCUSSION THREAD

**Discussion by BKo1706** [heig-vd-dai-course/heig-vd-dai-course#349] [Show and tell]
We will add functions to crypt or uncryt files based on privates keys.
Another option will allow us to generate a new private key which will be use for the encryption.

Submission (20.10.24, 20h30): 
repo link : https://github.com/BKo1706/DAI-07-practical-work-1
commit hash : b0bc76e6fe28f439ba4d7d8d7e7ad97b9f678e07

**Comment by ludelafo** [comment]
Looks good to us, thank you!
